src/civic_canister_frontend/scripts/issue-credential.ts

```diff
@@ -29,15 +29,15 @@ const alumniOfClaim = {
 };
 const mixedClaim = {
   claims: [
-    ["Is over 18", { Boolean: true }],
-    ["name", { Text: "Max Mustermann" }],
-    ["alumniOf", { Claim: alumniOfClaim }]
+    ["tigoYhp9SpCDoCQmXGj2im5xa3mnjR1zuXrpCJ5ZRmi", { Boolean: true }],
+    ["Status", { Text: "ACTIVE" }],
+    ["expirationDate", { Number: 1000 + 365 * 24 * 60 * 60 * 1000 }]
```

Comment:
I would expect something relative here, e.g. currentData + constant but I guess this is just for demoing anyway?